contracts/ethereum_contracts.go

```diff
@@ -1287,6 +1288,24 @@ func (v *EthereumKeeperRegistry) RegisterUpkeep(target string, gasLimit uint32,
 	return v.client.ProcessTransaction(tx)
 }
 
+// CancelUpkeep cancels the given upkeep ID
+func (v *EthereumKeeperRegistry) CancelUpkeep(id *big.Int) error {
```

Comment:
does this function get used anywhere?

Reply:
Used in https://github.com/smartcontractkit/chainlink/pull/6618 in a different repo :upside_down_face: